dev_guide/builds.adoc

```diff
@@ -1734,6 +1734,26 @@ Providing both `*script*` and `*command*` simultaneously creates an invalid
 build hook.
 ====
 
+==== Using the command line
```

Comment:
Please add the following anchor above the title:
[[builds-using-the-command-line]]